.github/workflows/Lib edFi.common pullrequest.yml

```diff
@@ -18,11 +18,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-    - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
-      with:
-        dotnet-version: 6.0.x
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4    
```

Comment:
The step was partially removed.

Reply:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/actions/runs/7466652036/job/20318481037


(https://ed-fi-alliance.slack.com/archives/D04G1UMG3BM/p1704993915723789)
Reason I have this     - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4

Becuase  I need to checkout ODS repo for Restore NuGet packages step